### ISSUE TITLE
Phison 2251-07

### ISSUE THREAD

**Issue by B00sterman** [brandonlw/Psychson#111]
Hi,
I know this has been posted multiple times, but I would like to test this vulnerability, whether with this script or not. I've got a Patriot Supersonic Xpress 16 GB, and, of course, I was gifted with the 2251-07 version. Is there ANY possibility to get this working on the 07?

**Comment by xeanarth** [comment]
i follow, same as you need that info :)

**Comment by B00sterman** [comment]
Yep :/ Doesn't seem like it tho. I am familiar with C++ etc, but only application coding. I don't know much about binaries applying to other chip types, like the 8051 one used here...

**Comment by xeanarth** [comment]
can we have a private talk? my mail is leox.howl@gmail.com . 
Send me a mail or share your here :D

**Comment by ghost** [comment]
Same here it sucks, I wish there was an update. I could help as well, I know a bit of c++

**Comment by haaha123123123** [comment]
selling 2251-03/2303 chips flash disk,contact me if you want buy
I have lots of 2251-03/2303 flsh disk,and it works with badusb,if anyone want to buy,please contact me, my email is haaha123123@126.com

**Comment by lotsa-spaghetti** [comment]
GitHub is not a selling platform. Advertise your offer/scam somewhere other
like eBay.
Am 09.02.2016 03:05 schrieb "haaha123123123" notifications@github.com:
